pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -455,3 +461,36 @@ func fixLibvirtCrcNetworkActive() (bool, error) {
 	}
 	return true, nil
 }
+
+func checkCrcDnsmasqConfigFile() (bool, error) {
+	c := []byte(crcDnsmasqConfig)
+	_, err := os.Stat(crcDnsmasqConfigPath)
+	if err != nil {
+		return false, fmt.Errorf("File not found: %s: %s", crcDnsmasqConfigPath, err.Error())
+	}
+	config, err := ioutil.ReadFile(crcDnsmasqConfigPath)
+	if err != nil {
+		return false, fmt.Errorf("Error opening file: %s: %s", crcDnsmasqConfigPath, err.Error())
+	}
+	if !bytes.Equal(config, c) {
+		return false, fmt.Errorf("Config file contains changes: %s", crcDnsmasqConfigPath)
+	}
+	return true, nil
+}
+
+func fixCrcDnsmasqConfigFile() (bool, error) {
+	cmd := exec.Command("sudo", "tee", crcDnsmasqConfigPath)
+	cmd.Stdin = strings.NewReader(crcDnsmasqConfig)
+	buf := new(bytes.Buffer)
+	cmd.Stderr = buf
+	if err := cmd.Run(); err != nil {
+		return false, fmt.Errorf("Failed to write dnsmasq config file: %s: %s: %v", crcDnsmasqConfigPath, buf.String(), err)
+	}
+
+	sd := systemd.NewHostSystemdCommander()
+	if _, err := sd.Restart("NetworkManager"); err != nil {
```

Comment:
This is still not fixed we just need to reload the `NetworkManager` service.

Reply:
huh? Oh chips... I think I know what happened. I didn't save this file from Sublime. It was edited, but uncommitted... pushing asap.